enterprise/frontend/src/metabase-enterprise/tools/ErrorOverview.jsx

```diff
@@ -108,7 +109,7 @@ export default function ErrorOverview(props) {
               sorting.column,
               getSortOrder(sorting.isAscending),
             )}
-            className="mt2 bounded-overflow-x-scroll"
```

Comment:
Was that a non-existing class? 😕

Reply:
Yeah. I searched the whole codebase but this class is mentioned nowhere